build.sh

```diff
@@ -1,15 +1,15 @@
-#!/bin/bash
+#!/bin/bash -eu
 
-set -eu
+# Make sure we are in the directory this script is in.
+cd "$(dirname "$0")"
```

Comment:
I think in this case, clarity is more important than brevity or speed. I mean, look at it, `${0%/*}`? It's horrible!